tests/unit/hooks/test_strtobool.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+
+from airflow_clickhouse_plugin.hooks.clickhouse import strtobool
+
+
+class StrToBoolTestCase(unittest.TestCase):
+    def test_correct_true(self):
+        self.assertTrue(strtobool('true'))
+
+    def test_correct_one(self):
+        self.assertTrue(strtobool('1'))
+
+    def test_correct_false(self):
+        self.assertFalse(strtobool('false'))
+
+    def test_correct_zero(self):
+        self.assertFalse(strtobool('0'))
```

Comment:
A quick best practice comment: the tests must cover all the supported input values, not only a few. A full coverage is required. However, this is a boilerplate code: you may use `self.subTest` functionality to check all truthy and all falsy values in a loop instead of creating a test per value.

Reply:
ok, fixed